### PR TITLE
Update gocql dependency

### PR DESCRIPTION
**Pull request by dmitsh**
# Description

Updated gocql dependency

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #691

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by berndverst — APPROVED** [review]
LGTM

**Comment by berndverst** [comment]
@dapr/approvers-components-contrib one of you please also review and merge. Thanks!

**Comment by artursouza** [comment]
@dmitsh Please, create a PR updating dapr/dapr with the latest version of dapr/components-contrib after this is merged.

**Review by artursouza — APPROVED** [review]

**Pull request merged**
